lib/LedgerSMB/Tax.pm

```diff
@@ -47,7 +47,7 @@ sub init_taxes {
     my @accounts = split / /, $taxaccounts;
     if ( defined $taxaccounts2 ) {
         #my @tmpaccounts = @accounts;#unused var
-        $#accounts = -1;# empty @accounts,@accounts=();
+        @accounts=(); # empty @accounts
```

Comment:
@ehuelsmann can you double check the original code and new here.
I'm wondering if there may perhaps have been a good reason it was done the way it was (after all the comment show its equivalent to the new code so why didn't the code it's self get changed instead of adding the comment)